src/containers/shared/css/table.scss

```diff
@@ -61,3 +62,33 @@ table.basic {
     text-align: center;
   }
 }
+
+table.token-table {
+  margin-top: 24px;
+  font-size: 14px;
+
+  .col1 {
+    width: 180px;
+    padding-bottom: 8px;
+    color: $gray-400;
+    text-transform: uppercase;
+    @include semibold;
+  }
+
+  .col2 {
+    padding-bottom: 8px;
+    color: $white;
+
+    a {
+      color: $green;
+      text-decoration: none;
+    }
+
+    .copy {
+      position: absolute;
+      width: 17px;
+      height: 17px;
+      margin-left: 8.42px;
```

Comment:
is this coming from the design? seeing fractional pixels doesn't make sense to me. i don't even know what that would mean. looking around, this is going to get rounded anyway: https://stackoverflow.com/questions/9080633/can-a-css-pixel-be-a-fraction

Reply:
yeah i got 8.42 by cursoring on the figma, I guess i will  just put it a 8px then